### PR TITLE
Clang-formatting the Database code

### PR DESCRIPTION
**Pull request by schmidt-sebastian**
This will allow us to run ./scripts/style.sh on any pending changes.

### CODE REVIEW

File: Firebase/Database/Utilities/Tuples/FTupleOnDisconnect.h [hunk 1]
```diff
@@ -14,14 +14,14 @@
  * limitations under the License.
  */
 
-#import <Foundation/Foundation.h>
 #import "FTypedefs_Private.h"
+#import <Foundation/Foundation.h>
```
Comment (paulb777) [inline Firebase/Database/Utilities/Tuples/FTupleOnDisconnect.h]: Consider putting Foundation.h in its own group - separated by spaces

File: Example/Database/Tests/Helpers/FDevice.m [hunk 2]
```diff
@@ -14,120 +14,131 @@
  * limitations under the License.
  */
 
-#import <XCTest/XCTest.h>
 #import "FDevice.h"
-#import "FirebaseCommunity/FIRDatabaseReference.h"
-#import "FRepoManager.h"
-#import "FIRDatabaseReference_Private.h"
+#import <XCTest/XCTest.h>
```
Comment (wilhuff) [inline Example/Database/Tests/Helpers/FDevice.m]: Please put system imports in a separate group.

File: Example/Database/Tests/Helpers/FTestBase.m [hunk 3]
```diff
@@ -14,157 +14,182 @@
...
  */
 
-#import "FirebaseCommunity/FIRApp.h"
-#import "FirebaseCommunity/FIROptions.h"
 #import "FTestBase.h"
-#import "FTestAuthTokenGenerator.h"
 #import "FIRDatabaseQuery_Private.h"
 #import "FIRTestAuthTokenProvider.h"
+#import "FTestAuthTokenGenerator.h"
+#import "FirebaseCommunity/FIRApp.h"
```
Comment (wilhuff) [inline Example/Database/Tests/Helpers/FTestBase.m]: You might consider splitting these out too.

File: Example/Database/Tests/Integration/FData.m [hunk 4]
```diff
@@ -15,2673 +15,3100 @@
  */
 
 #import "FData.h"
-#import "FTestHelpers.h"
+#import <limits.h>
```
Comment (wilhuff) [inline Example/Database/Tests/Integration/FData.m]: system headers should definitely be separate.

**Comment by schmidt-sebastian** [comment]
I am going to hold off on these changes for now since they break the compile. Most of this is probably due to the issues pointed out above and can be fixed by properly arranging the headers.

**Comment by wilhuff** [comment]
Likely the problem is that the various files are getting some of their dependencies transitively rather than including everything they use.

**Pull request closed without merging**